app/jobs/workflow_execution_cancelation_job.rb

```diff
@@ -4,6 +4,33 @@
 class WorkflowExecutionCancelationJob < ApplicationJob
   queue_as :default
 
+  # When server is unreachable, continually retry
+  retry_on Integrations::ApiExceptions::ConnectionError, wait: :polynomially_longer, attempts: Float::INFINITY
+
+  # Puts workflow execution into error state and records the error code
+  retry_on Integrations::ApiExceptions::APIExceptionError, attempts: 3 do |job, exception|
```

Comment:
**nitpick:** Can we set the `wait` for this as by default it appears to be `3.seconds` (https://github.com/phac-nml/irida-next/pull/546/files)

I'm not sure is `15.seconds` is a good choice or if we set it to `:polynomially_longer` with `attempts: 3`. Since that would be (first wait ~3s, then ~18s, then ~83s, etc).

Reply:
Changed in https://github.com/phac-nml/irida-next/pull/546/commits/ebfe278e975461c66edf27a7560aad0a453916eb

I opted for `:polynomially_longer`